questionnaire/questionnaire-service.js

```diff
@@ -167,7 +170,10 @@ function createQuestionnaireService({
                 }
 
                 if (task.type === 'sendSms') {
-                    if (pointer.has(questionnaire, task.templatePlaceholderMap.phoneNumber)) {
+                    if (
+                        pointer.get(questionnaire, task.templatePlaceholderMap.phoneNumber) !==
```

Comment:
As above, can we cache the value as per the replace-json-pointer "has" check?

Reply:
np